src/OpenLoco/src/Ui/ViewportInteraction.cpp

```diff
@@ -921,6 +921,10 @@ namespace OpenLoco::Ui::ViewportInteraction
                     {
                         interactionsToExclude |= InteractionItemFlags::building | InteractionItemFlags::headquarterBuilding | InteractionItemFlags::industry;
                     }
+                    if (vp->hasFlags(ViewportFlags::seeThroughBridges))
```

Comment:
Bridges can't be interacted with so it makes sense that this does nothing but i feel writing: `interactionsToExclude |= InteractionItemFlags::none;` which compiles to a no-op isn't great. Can you just change this to a comment.
`// Bridges can't be interacted with ever so no need to exclude anything additional`

Reply:
Done